scripts/test.sh

```diff
@@ -9,7 +9,7 @@ CONFIG_FILE_NAME="hardhat.config.ts"
 
 # setup example repo
 rm -rf starknet-hardhat-example
-git clone -b plugin --single-branch git@github.com:Shard-Labs/starknet-hardhat-example.git
+git clone -b relativeArtifacts --single-branch git@github.com:Shard-Labs/starknet-hardhat-example.git
```

Comment:
TODO: change before merge

Reply:
@epShardlabs You can change this back to `plugin` as your example repo PR has been merged.